pkg/strc/middleware_echo_test.go

```diff
@@ -139,3 +142,116 @@ func TestEchoMiddlewareFilter(t *testing.T) {
 		t.Errorf("Response body not as expected: %s", rec.Body.String())
 	}
 }
+
+func TestEchoRequestLoggerFilter(t *testing.T) {
+	logHandler := collect.NewTestHandler(slog.LevelDebug, false, false, false)
+	logger := slog.New(logHandler)
+
+	e := echo.New()
+	e.Use(strc.EchoRequestLogger(logger, strc.MiddlewareConfig{
+		Filters: []strc.Filter{strc.IgnorePathPrefix("/metrics")},
+	}))
+	e.GET("/metrics", func(c echo.Context) error {
+		return c.String(http.StatusOK, "metrics")
+	})
+
+	req := httptest.NewRequest(http.MethodGet, "http://example.com/metrics", nil)
+	rec := httptest.NewRecorder()
+	e.ServeHTTP(rec, req)
+
+	assert.Equal(t, 0, logHandler.Count())
+}
```

Comment:
Please completely remove filtering feature as well, turns out we do not need it. A patch in CRC that sets middleware per-group was just put to merge queue and it is just bunch of code that we would need to maintain.